nodebuilder/tests/swamp/swamp.go

```diff
@@ -103,8 +104,10 @@ func NewSwamp(t *testing.T, options ...Option) *Swamp {
 
 	swp.t.Cleanup(func() {
 		swp.stopAllNodes(ctx, swp.BridgeNodes, swp.FullNodes, swp.LightNodes)
-		cleanupCoreNode()
-		cleanupGRPCServer()
+		err = cleanupCoreNode()
+		assert.NoError(t, err)
```

Comment:
require 🙏🏻 (for both)

Reply:
yeah sure thing https://github.com/celestiaorg/celestia-node/pull/1389/commits/a531f6b2f1ba165eb66d3232dca3ad38c03d5916

for my own understanding, what is reasoning of using require over assert here?